ci/est/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# defaults file for est
+
+shared_workspace: /tmp/workdir/pki
+github_workspace: ./
+
+#DS
+ds_container: ds
+ds_image: quay.io/389ds/dirsrv
+ds_hostname: ds.example.com
+ds_password: Secret.123
```

Comment:
> Is it possible to override these values from command line in case we want to use different settings, or do we have to edit the file?

Yes, there are _role_ default values and have very lower precedence: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

They can be overwritten in many other file and in the command line as well